dtsi.py

```diff
@@ -35,6 +77,10 @@ def generate_panel_dtsi(p: Panel, options: Options) -> None:
 ''')
 		f.write(f'''\
 &mdss_dsi0 {{
+	pinctrl-0 = <&mdss_default>;
+	pinctrl-1 = <&mdss_sleep>;
+	pinctrl-names = "default", "sleep";
```

Comment:
Hm, we also generate this if there is no `reset` in `options.gpio`. The reset gpio is usually there but this is not guaranteed. e.g. samsung-gtelwifiue in linux-panel-drivers.